### ISSUE TITLE
x0/y0 are cast to x0w and y0w but never used

### ISSUE THREAD

**Issue by antevens** [sparkfun/HyperDisplay_SSD1309_ArduinoLibrary#2]
### Minor compilation warning

x0w and y0w seem unused here, did you mean to use them below?

```
232     color_t value = data; // have to skip color sequences in buffered writing mode
233
234     hd_hw_extent_t x0w = (hd_hw_extent_t)x0; // Cast to hw extent type to be sure of integer values
235     hd_hw_extent_t y0w = (hd_hw_extent_t)y0;
236
237     SSD1309_Bite_t user = *((SSD1309_Bite_t*)value);
238     if(user.b0){ // Check if the user's bit is set or not (this implies that the user should always set bit 0 of a 'bite' to the pixel value they want)
239         // Need to set the pixel high
240         setWindowPixel(x0, y0, pCurrentWindow);
241     }else{
242         // Need to clear the pixel
243         clearWindowPixel(x0, y0, pCurrentWindow);
244     }
245     // updateRefreshZone( x0, x0, y0, y0);  // Tell where we need updates
246     // refreshDisplay();                    // Perform updates
247 }
```

**Comment by oclyke** [comment]
Spot on! Thanks. 

How is the library working for you? I'd love to hear any more suggestions you might have.

**Comment by antevens** [comment]
Hey Owen, 

Thanks for your hard work, I'm currently using this library for a part of a hobby project I'm doing to make a HUD/Bike computer for my bike helmet/track bike, I got frustrated with my Garmin bike computer because it could only do 1 second resolution on it's lap timing while I needed at least millisecond resolution and I figured the best way to show the data was a HUD in my helmet.

I picked up an LCD-15173 and LCD-14606 to use for testing, one thing that I did consider was adding support for LCD-14606 in Hyperdisplay, seems like it shouldn't be to hard because the controller is very similar to LCD-15173. Still don't know which display I'll use, the LCD-15173 seems a bit dangerous in case of a crash since it's glass and I don't yet know how useful it is without the ability to focus at infinity.

For now I'm working on the Bluetooth part (I'm working with a Nano 33 BLE), once I've got all the data sources and logging sorted I'll put focus back on the display aspect and I'll probably come across more things and make suggestions.

A photo to give you an idea of how the visor will look with a display:
https://photos.app.goo.gl/TtnNNLjsGLr2yVZq7

The whole helmet looks like this:
https://media-cdn.canyon.com/dw/image/v2/BCML_PRD/on/demandware.static/-/Sites-canyon-master/default/dw6307b9ff/images/full/9100330_GI/2019/9100330_GIRO_Aerohead_Mips_bk_gy.png

https://github.com/antevens/dromedary/blob/master/hud/sensors/sensors.ino
